src/com/tms/lesson11/AddToCollection.java

```diff
@@ -0,0 +1,42 @@
+package com.tms.lesson11;
+
+import java.util.List;
+
+public class AddToCollection implements Runnable {
```

Comment:
please rename this class, method should use verb in name, not class

Reply:
fix